setup.py

```diff
@@ -51,7 +51,7 @@ def main():
         },
         scripts=[],
         zip_safe=False,
-        install_requires=[],
+        install_requires=['requests', 'Pillow'],
```

Comment:
If you checkout how we are handling the optional requirement defusedxml, that should give you a great place to start. The key is, in your mixin, you'll want to do something like:

```python
try
    import requests
    from PIL import Image
except ImportError:
    use_resize_mixin = False
else:
    user_resize_mixin = True
```

Then, at the entry point for the mixin check to see what the value of that boolean flag is and early exit if those two libraries are missing. Does that make sense?

I would be more than happy to help you if you would like. I'm pretty excited about this mixin.